### DISCUSSION TITLE
Is the fire_emis_nl in drv_flds_in ever populated or actually used in experiments?

### DISCUSSION THREAD

**Discussion by mvertens** [ESCOMP/CTSM#2397] [General]
@wwieder @dlawrenncar @fvitt  - I am  wondering if the drv_flds_in namelist setting for fire_emis_nl is actually every used in experiments. CLM is responsible for setting this namelist in drv_flds_in, however I could not see any call in CLM's buildnml to CLMBuildNml.pm that would actually send the fire-emis flag.

In addition I looked through all of the B compset simulations in /glade/p/cesmdata/cseg/runs/cesm2_0 and could not see this activated in any cesm2 or cesm3 development simulations. Is this a feature that is just there but never activated?  I am wondering if this flux is ever passed from CLM to the CAM and then used in the CAM chemistry. Is there an intention to turn this on in future simulations?

**Comment by dlawrenncar** [comment]
We have run with interactive fire aerosol emissions previously and currently, @lawrencepj1 is running some test simulations with fire aerosol emissions for the NSF Growing Convergence project that he has.  I don't think any production runs have ever been done with this on.  However, we are considering, lightly, whether or not we could run with interactive fire aerosol emissions for CESM3.  One of the main issues for turning it on relates to spinup.

**Reply by mvertens** [reply]
@dlawrenncar - thanks for the quick response. We have prioritized having an implementation in FATES too allow interactive fire aerosol emissions and have it also with the oslo-aero chemistry implementation in CAM/NORESM. In looking at the CLM namelist settings and how CAM/OSLO utliizes this - it would never work with the current settings which made me wonder how this worked in the current simulations.

**Reply by ekluzek** [reply]
@mvertens you turn it on by adding "-fire_emis" to the CLM_BLDNML_OPTS XML setting. We have that on for most of our non-FATES tests. We turn it off when FATES is on, because they don't work together as you know (see #1045). I could look more into the details of how this works, but a case from CAM-Chem people or Peter sounds better.

By the way when coupled to CAM -- CAM turns the namelist in. So CAM-Chem cases sets it up.

We are excited that you'll be working on this since this has been an outstanding issue for FATES in CTSM that we'd like to see working. It's one of the important requirements that we need to use FATES in the fully coupled system.

**Reply by mvertens** [reply]
@ekluzek - thanks for the details. That's very helpful. I was surprised that I could not see it activated in any of the production runs - so having feedback from @lawrencepj1 and pointers to his simulations would be really helpful as well.

**Comment by dlawrenncar** [comment]
Got it.  I would suggest asking @lawrencepj1  to point you to his cases that are running with interactive fire aerosols.

**Reply by lawrencepj1** [reply]
Thanks @dlawrenncar , @mvertens and @ekluzek,

Yes, as Dave says I have CESM2.1.5 running with active fire and MEGAN BVOC aerosols without CAM Chem. The model takes the outputs of the fire model and MEGAN and then applies some scalars to replaces the values read in from the CMIP6 forcing files used by MAM4. This requires a number of things such as turning on -fire_emis and -megan as clm_bldnml_opts values in the env_run.xml file. It also requires specifying the emissions in the user_nl_cam and user_nl_clm files. To process the new expressions the shr_expr_parser_mod.F90 needs to be updated in the cime drivers. The changes are made in my github account

https://github.com/lawrencepj1/CESM2.1.5-CIME-Active-BVOCs/compare/master...baselines

There are also some changes to clm5 to deal with the new fire emission factors

https://github.com/lawrencepj1/CESM2.1.5-CLM5-New-Land-Use/compare/master...active-bvocs

The instructions for downloading the code and building the case are in 
[BaselineReferenceFireBVOCsSpinUpCaseSetup.docx](https://github.com/ESCOMP/CTSM/files/14487364/BaselineReferenceFireBVOCsSpinUpCaseSetup.docx)

The case is on
/glade/work/lawrence/cesmccisens2024/cases-baselines/b.e21.B1850.f09_g17.ccisens-reference-firebvocs.001

Output is being archived to
/glade/campaign/cgd/tss/lawrence/ccisensarchive/b.e21.B1850.f09_g17.ccisens-reference-firebvocs.001

It has been a struggle to spin up the full model. The default values provided by Simone Tilmes produced very strong cooling with a negative restom. I have now scaled conversion factors to match the CMIP6 forcing files which alleviates much of the cooling but the new radiation regime impacts the carbon cycle which I am in the process of spinning back into balance. Not simple but possible.

Best
Peter

**Comment by mvertens** [comment]
@lawrencepj1 - thanks for this. this is super helpful.

**Comment by ekluzek** [comment]
I'm going to close this discussion because it seems to be resolved. It's still there though and can be reopened.
